docs/getstarted.md

```diff
@@ -412,3 +412,68 @@ fig.update_layout(
     title='Loss vs epochs'
 )
 ```
+
+## Testing new data
+
+If new PDB files need to be tested with a pre-trained model, the first step would be to process and save them into HDF5 files. Let's suppose that the model has been trained with `ProteinProteinInterfaceResidueQuery` queries mapped to graphs:
```

Comment:
Added to the readme. I think it will be nice to add a tutorial only when we publish a pre-trained model (such as the one used in the 3D-Vac paper), or when we create a more "advanced" tutorial. For now having that in the .md files only is fine.

Reply:
agreed